projects/ngrx.io/content/guide/store/feature-creators.md

```diff
@@ -124,6 +124,36 @@ export const booksFeature = createFeature({
 
 The `extraSelectors` option accepts a function that takes the generated selectors as input arguments and returns an object of all the extra selectors. We can use it to define as many extra selectors as we need.
 
+### Reusing Extra Selectors
+
+Reusing extra selectors can be done by defining `extraSelectors` factory in the following way:
+
+<code-example header="books.feature.ts">
+export const booksFeature = createFeature({
```

Comment:
Thanks for your contribution!
Could you also include the imports and the state interface/constant here please, that way the snippets show the bigger picture and it keeps things consistent with the other snippets.

Reply:
Thank your for your review. I included the missing parts. Please recheck.